soc/arm64/nxp_imx/mimx9/linker.ld

```diff
@@ -1,8 +1,354 @@
 /*
- * Copyright 2022 NXP
+ * Copyright (c) 2013-2014 Wind River Systems, Inc.
```

Comment:
Unless this is essentially a full replacement, removing an old copyright might not be a good idea. I'd say at least a bit of discussion is warranted.

Reply:
It's basically a copy/paste of the arm64's linker script with some SOF additions. I guess due to these additions we can keep the copyright. Will fix.